mobile.css

```diff
@@ -0,0 +1,110 @@
+.body {
+    max-width: 375px;
+    margin: 0;
+}
+.nav__link {
+    display: none;
+}
+.logo {
+    width: 138px;
+    height: 42px;
+    margin: 5px 2px;
+}
+.vectors {
+    display: none;
+}
+.nav {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 375px;
+    margin-bottom: 30px;
+}
+.burger {
+   width: 24px;
+   height: 15px;
+   margin-right: 21px;
+}
+.title {
+    max-width: 206px;
+}
+.title__h1 {
+    margin: 0;
+    font-size: 50px;
+}
+.title__h2 {
+    font-size: 10px;
+    width: 161px;
+    margin: 0 auto;
+}
+.image__background {
+    width: 448px;
+    height: 139px;
+    margin-left: -30px;
+    margin-top: 8px;
+    margin-bottom: 86px;
+}
+.image__car {
+    width: 357px;
+    height: 166px;
+    margin-left: 9px;
+    margin-top: 120px;
+}
+.box {
+    margin: 0 auto 34px;
+    max-width: 303px;
+    flex-wrap: wrap;
+}
+
+.box__img-1 {
+    width: 38.14px;
+    height: 28.7px;
+}
+.box__img-2 {
+    width: 25.29px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)